internal/nginx/main.go

```diff
@@ -38,8 +37,8 @@ var TemplatePath = "/etc/nginx/template/nginx.tmpl"
 // PID defines the location of the pid file used by NGINX
 var PID = "/tmp/nginx.pid"
 
-// StatusSocket defines the location of the unix socket used by NGINX for the status server
-var StatusSocket = "/tmp/nginx-status-server.sock"
+// StatusPort port used by NGINX for the status server
+var StatusPort = 10256
```

Comment:
What's the reason you're switching to tcp?

Reply:
> What's the reason you're switching to tcp?

If the node is smaller than should be or during spikes of CPU, the request fails (socket). I cannot reproduce this consistently but when I can is only related to CPU utilization.

After this is merged I will add an e2e test so show how this can be reproduced (using [stress](https://linux.die.net/man/1/stress))

Reply:
@aledbf did this e2e test ever get added? I'm running into a similar situation when CPU spikes and I'm wondering if the switch away from a socket actually resolves it